### DISCUSSION TITLE
SVG source files with CMYK color profile

### DISCUSSION THREAD

**Discussion by brainchild0** [ImageMagick/ImageMagick#2902] [Help]
I have an SVG documented created in Inkscape that has an eciCMYK color profile. I wish to perform a rasterization effect on the image (tiling), with the objective of publishing the result to a PDF/X file, also having a CMYK color profile.

To test how ImageMagick would handle the source file, I began with a simple command:

```
$ convert test.svg test.tiff
```

The effect is that objects are rendered as entirely black in the target TIFF image.

I tried to improve the results by adding information about source and target color profiles:

```
$ convert test.svg  -colorspace CMYK -set colorspace CMYK test.tiff
```

In this case, unfortunately, the result is a solid black canvas.

Would ImageMagick offer support for the desired workflow, beginning with a demonstration of simply converting to a viable TIFF image with a CMYK color profile?

Note that in principle the ImageMagick effect might be applied to a TIFF representation of the original image that has been manually exported from Inkscape. Unfortunately, Inkscape has no support for writing exported files with such a color profile.

**Comment by snibgo** [comment]
What version of IM, on what platform? What SVG renderer does your IM use? Note that Inkscape is the best renderer.

Please link to a sample SVG with CMYK profile.
